pkg/handler/unrouted_handler.go

```diff
@@ -915,15 +915,8 @@ func (handler *UnroutedHandler) sendError(w http.ResponseWriter, r *http.Request
 		statusErr = NewHTTPError(err, http.StatusInternalServerError)
 	}
 
-	reason := append(statusErr.Body(), '\n')
-	if r.Method == "HEAD" {
-		reason = nil
```

Comment:
Do you know how `http.Error` handles HEAD requests? Does it include the body or not?